pkg/operator/ceph/controller/spec_test.go

```diff
@@ -171,6 +182,29 @@ fi`}
 	assert.Equal(t, livenessProbeTimeoutSeconds, probe.TimeoutSeconds)
 }
 
+func TestGenerateReadinessProbeExecDaemon(t *testing.T) {
+	daemonID := "a"
+	probe := GenerateReadinessProbeExecDaemon(config.MgrType, daemonID)
+	expectedCommand := []string{"env",
```

Comment:
if we change the error output, this might be changed accordingly

Reply:
Fixed